src/main/java/net/dries007/tfc/client/gui/GuiCustomizeWorld.java

```diff
@@ -126,6 +127,7 @@ public void initGui()
         GuiPageButtonList.GuiListEntry[] page1 = new GuiPageButtonList.GuiListEntry[] {
             new GuiPageButtonList.GuiSlideEntry(ID_SPAWN_FUZZ, I18n.format("createWorld.customize.custom.spawnfuzz"), true, this, 0, 2500, settings.spawnFuzz),
             new GuiPageButtonList.GuiButtonEntry(ID_FLAT_BEDROCK, I18n.format("createWorld.customize.custom.flatbedrock"), true, settings.flatBedrock),
+            new GuiPageButtonList.GuiButtonEntry(ID_SMALL_ROCK, I18n.format("createWorld.customize.custom.smallrocklayer"), true, settings.enableSmallRocklayerSize),
```

Comment:
Is there a reason this can't be a slider from say 1-10?

Reply:
Yeah no reason, although 1 is ridiculously small and anything larger than 6 is ridiculously big. The only real issue that ever arises is in SSP and not finding all the rocks needed. So just offering the single option seemed less likely to cause further Issues or complaints.